src/scss/components/header/_cart.scss

```diff
@@ -0,0 +1,205 @@
+
+.header__cart {
+  float: right;
+
+  .cart-info, &-btn, .shopping-cart {
+    display: none;
+  }
+
+  &-icon {
+    display: inline-block;
+    height: 30px;
+    position: relative;
+
+    svg {
+      fill: $gray;
+      height: 30px;
+      width: 30px;
+      vertical-align: middle;
+    }
+
+    .cart-products-number {
+      color: $tertiary;
+      font-family: $franklin-medium-cd;
+      font-size: 12px;
+      left: 9px;
+      line-height: 1;
+      position: absolute;
+      text-align: center;
+      top: 5px;
+      width: 14px;
+    }
+  }
+}
+
+@media (min-width: 48em) {
+  .header__cart {
+    $root: &;
+    position: relative;
+
+    &-wrapper {
+      align-items: center;
+      background: $white;
+      border: 1px $gallery solid;
+      border-radius: $globalBorderRadius;
+      display: flex;
+      flex-direction: row;
+      padding: 9px;
+      position: relative;
+      z-index: 10;
+    }
+
+    &:hover, &:focus {
+      #{$root}-wrapper {
+        border-bottom-width: 0;
+        border-radius: $globalBorderRadius $globalBorderRadius 0 0;
+        border-color: $tertiary;
```

Comment:
I propose to create variables for each component. Or use some pre-defined variables. For border we have set of $borderTertiaryColor...ets.